.github/workflows/crowdin-download.yml

```diff
@@ -0,0 +1,56 @@
+name: Create Automated Crowdin Translations PR
+
+on:
+  schedule:
+    - cron: '0 0 * * 3' # 00:00 UTC on Wednesday
+  workflow_dispatch: # Allow the workflow to be manually dispatched.
```

Comment:
This hasn't run against the PR yet. Should we wait until Wednesday to validate it before merging?

Reply:
I usually test github actions by creating a PR. Will it not run because it's scheduled?

Reply:
Yep, this is a scheduled/manual pipeline, so it won't execute when this PR changes.